client-js/src/events.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  CLValueParsers,
+  CLMap,
+  CLString,
+  CLValueBuilder,
+  CLTypeTag,
+  CLValue,
+  CasperServiceByJsonRPC,
+} from "casper-js-sdk";
+
+import { Parser } from "@make-software/ces-js-parser";
+
+import { EventItem, EventParsed, CEP47Events, Transform } from "./types";
+
+export const CEP47EventParserFactory =
```

Comment:
It basically modified version of [this](https://github.com/casper-network/casper-contracts-js-clients/blob/master/packages/cep47-client/src/index.ts#L46-L101)